clients/admin-ui/src/features/common/custom-fields/types.ts

```diff
@@ -9,3 +14,33 @@ export interface CustomFieldWithIdExtended extends CustomFieldWithId {
 export interface Tab extends TabData {
   submitButtonText: string;
 }
+
+export interface CustomFieldDefinitionExisting
```

Comment:
`...Existing` is the best I could think of. The `...WithId` type allows `id` to be `undefined` which isn't useful in most of the code.